index.html

```diff
@@ -206,33 +237,37 @@ <h2 class="professional-text"> Profesional Art Printing Data </h2>
         <li class="card3">Ruby</li>
       </ul> 
 
-      <button class="button2" type="button">
+      <button class="button2 buttontwo" type="button">
         <a class="button-text2" href="#"> See Project </a>  
     </button> 
       </div>
+    </div>
+  
     </section>
+  </section> 
 
       <!-- about me section -->
 
      <section class="about-me">
+      <div class="about-me-div">
       <h2 class="about-me-text">About me</h2>
       <p class="about-me-para">Hello I'm a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't hestiate to contact me.</p>
 
```

Comment:
- [ ] I beleive this paragraph should have the font-size 20px from the figma design, currently it is on 14px and looks a bit different from the original design, how about we fix this :+1: